darts/models/torch_forecasting_model.py

```diff
@@ -249,11 +250,17 @@ def fit(self,
             covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
             val_series: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
             val_covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
-            verbose: bool = False) -> None:
+            verbose: bool = False,
+            epochs: int = 0) -> None:
```

Comment:
I was thinking about something like it at first, but then realized that it does not make much sense to have `n_epochs` in the constructor, because number of epochs **to** train should not be a property of the whole model, but merely a parameter to `fit()` method. Same approach is used for example in Keras.
So I was hoping to leave `epochs` here and convince you to remove `n_epochs` from the constructor entirely. I didn't wanted to do it in this PR though: all other torch models would have to have the same update.

Reply:
I agree, makes sense. Do you think after this one you could refactor the other torch models to do like you say?

Reply:
Concerning this treatment of epochs: I think we could *always* consider that calling fit() or fit_from_dataset() trains for *additional* epochs. I.e., the only place where we would re-initialise things would be in the constructor. WDYT?

Reply:
Great idea. I'll log an issue in JIRA.

Reply:
Concerning re-initialising only in the constructor: would this be part of the present PR? I think it's very linked to the present changes you are doing, so it would be nice if it is. Concerning adding the "epochs" to all models, I agree we can log an issue and do it later.

Reply:
Per discussion on slack, this will be fixed in another PR.